docs/public/hexclave-cli-template.py

```
@@ -0,0 +1,58 @@
+    raise Exception("project_id is required")
+
+  def post(endpoint, json):
+    headers = {
+      'Content-Type': 'application/json',
+      'x-hexclave-project-id': project_id,
+      'x-hexclave-access-type': 'client',
+    }
+    if publishable_client_key is not None:
+      headers['x-hexclave-publishable-client-key'] = publishable_client_key
+
+    return requests.request(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)